.github/workflows/ci.yml

```diff
@@ -92,3 +92,38 @@ jobs:
 
       - name: Show Cache
         run: du -h ${{ github.workspace }}/.cache/* || true
+
+  release-pr:
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
+          package-name: ipfs-companion
+          changelog-notes-type: github
+          command: release-pr
+
+  release-assets:
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/download-artifact@v3
+        id: download
+        with:
+          name: built-on-ubuntu-latest
+          path: ~/downloads/
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: ${{ steps.download.outputs.download-path }}/*.zip
+          fail_on_unmatched_files: true
+          generate_release_notes: true
+          draft: true
+          append_body: true
```

Comment:
`built-on-ubuntu-latest.zip`is nice,  but having to unpack it every time to get firefox or chrome specific version and then re-upload it to release is a bit tedious.

Mind adding a step that unzips it, so CI attach three `ipfs_companion*.zip` to the release, and person making the release does not need to unzip anything?